spec/requests/case_contacts_spec.rb

```diff
@@ -1,10 +1,10 @@
 require "rails_helper"
 
 RSpec.describe "/case_contacts", type: :request do
-  let(:organization) { build(:casa_org) }
+  let(:organization) { create(:casa_org) }
```

Comment:
why?

Reply:
To enable the organization register to be created into the database, so we can redirect_to casa_case#show

Reply:
does it break using build?

Reply:
Thank you for the tip. It wasn't really necessary. 👍🏼